pkg/services/secrets/kvstore/cache.go

```diff
@@ -0,0 +1,69 @@
+package kvstore
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+type decryptionCache struct {
+	decryptionCache map[string]*cachedDecrypted
+	sync.Mutex
```

Comment:
Don't seem to use this so this cache would probably create problems being accessed from multiple goroutines at the same time.